code/modules/power/cable.dm

```diff
@@ -441,6 +441,8 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list(new/datum/stack_recipe("cable restrain
 /obj/item/stack/cable_coil/proc/check_menu(mob/living/user)
 	if(!istype(user))
 		return FALSE
+	if(!iscyborg(user) && !ishuman(user))

```

Comment:
Why not a dexterity check?

Reply:
Will a dex check work on borgs?

Reply:
Oh god.  I feel embarsed.  I don't know the mob code very well and didn't know about the 	IsAdvancedToolUser check.